.gitlab-ci.yml

```diff
@@ -0,0 +1,34 @@
+image: 'rust:latest'
```

Comment:
Does this guarantee that we are on stable? How do toolchain updates get propagated?

Reply:
The latest tag is a relative tag. It always has the latest stable.

Reply:
Do we depend now on docker to update it or rustup? I would rather install the toolchain each time that way we can reduce possible issues.

Reply:
The rust team updates the docker image. https://hub.docker.com/_/rust/

I'm ok with not using the latest tag and specifying the latest stable, but not manually installing rustup and configuring all that when it is already done in their official image.

Reply:
Sure, I would say tho if the toolchain is already installed and rustup is also already installed the `rustup install stable` should be very quick.

Reply:
But to what end? When a new Rust stable version comes out, we just bump the image tag.

Reply:
You would rather do that manually? I would like this to be automated and on release days its not a thing we have to think about. Im fine though keeping the image.

Reply:
No, I would rather use the latest tag which does exactly what you are asking for. =P

Reply:
Works for me!